packages/server/db/users.js

```diff
@@ -19,8 +20,80 @@ const getDoc = async (collection, uid) => {
   throw new Error(`Unable to find '${uid}' in '${collection}' collection`);
 };
 
+const phoneRegex = RegExp(
+  /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/,
+);
+
+const mentorSchema = Yup.object().shape({
+  email: Yup
+    .string()
+    .email()
+    .required('Email Required'),
+  name: Yup
+    .string()
+    .required('Name Required'),
+  timezone: Yup
+    .string()
+    .required('Timezone Required'),
+  phone: Yup
+    .string()
+    .matches(phoneRegex, 'Phone number is not valid'),
+  pronouns: Yup
+    .string(),
+  college: Yup
+    .string(),
+  avatar: Yup
+    .string()
+    .required('Avatar Required'),
+  bio: Yup
+    .string()
+    .required('Bio Required'),
+  major: Yup
+    .string(),
+  subjects: Yup
+    .array()
+    .required('Subjects Required'),
+  gradeLevels: Yup
+    .array()
+    .required('Grade Levels Required'),
+});
+const parentSchema = Yup.object().shape({
+  email: Yup
+    .string()
+    .email()
+    .required('Email Required'),
+  name: Yup
+    .string()
+    .required('Name Required'),
+  timezone: Yup
+    .string()
+    .required('Timezone Required'),
+  phone: Yup
+    .string()
+    .matches(phoneRegex, 'Phone number is not valid'),
+  pronouns: Yup
+    .string(),
+  avatar: Yup
+    .string()
+    .required('Avatar Required'),
+});
+const studentSchema = Yup.object().shape({
+  email: Yup
+    .string()
+    .email(),
+  name: Yup
+    .string()
+    .required('Name Required'),
+  subjects: Yup
+    .array()
+    .required('Subjects Required'),
+  grade: Yup
+    .number()
+    .required('Grade Required'),
+});
+
```

Comment:
Can you move these Yup schemas to another file?

Reply:
Can I import them from the client?

Reply:
No. It is a good idea, but we want to keep the client and server separate right now.

Something we can do in the future is create a “shared” folder of code and resources shared by the two components; however, for the time being, this duplication is manageable.

Reply:
Should I call it `schemas.js`?

Reply:
Perfect.